### DISCUSSION TITLE
part-ii/2018-48/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#270] [Question comments]
Part II 2018 Differential Geometry: Paper 2, Section II, I 

https://questions.tripos.org/part-ii/2018-48/

**Comment by asheshbati** [comment]
Anyone got a hint for the last part?

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(d) Curvature and torsion are locally constant, hence globally constant by compactness. We can also show that torsion is constantly zero. 


</details>
